python/tidy/servo_tidy/tidy.py

```diff
@@ -773,6 +818,7 @@ def get_file_list(directory, only_changed_files=False, exclude_dirs=[]):
         args = ["git", "ls-files", "--others", "--exclude-standard", directory]
         file_list += subprocess.check_output(args)
         for f in file_list.splitlines():
+            f = os.path.join(*f.split("/")) if sys.platform == "win32" else f
```

Comment:
Does this really throw an exception in Windows? I'm curious because we're working on paths relative to the servo directory. Doesn't python handle a path like `./servo/python` appropriately?